docs/src/components/ComponentDoc/ComponentDoc.tsx

```diff
@@ -137,28 +137,23 @@ class ComponentDoc extends React.Component<any, any> {
             </>
           </Flex.Item>
         </Flex>
-        <Grid columns="auto 300px">
-          <Flex column>
-            <Flex.Item>
-              <Ref innerRef={this.handleExamplesRef}>
-                <ExampleContext.Provider
-                  value={{
-                    activeAnchorName: activePath,
-                    onExamplePassed: this.handleExamplePassed,
-                  }}
-                >
-                  <ComponentExamples displayName={info.displayName} />
-                </ExampleContext.Provider>
-              </Ref>
-            </Flex.Item>
-
-            <Flex.Item>
-              <div style={exampleEndStyle}>
-                This is the bottom <Icon name="pointing down" />
-              </div>
-            </Flex.Item>
-
-            {/* TODO: bring back the right floating menu
+        <Grid columns="auto 300px" styles={{ justifyContent: 'normal', justifyItems: 'stretch' }}>
```

Comment:
Fixes this issue:

![image](https://user-images.githubusercontent.com/14183168/53878889-b82da200-4015-11e9-8d8e-976b95efc698.png)
https://stardust-ui.github.io/react/components/form